uiplib/Wallpaper.py

```diff
@@ -38,14 +40,14 @@ def set_wallpaper_windows(self, filename):  # pragma: no cover
 
     def set_wallpaper_osx(self, filename):  # pragma: no cover
         """Set a file as OSX Wallpaper."""
-        from appscript import app, mactypes  # use applescript modules
+        command = ("osascript -e 'tell application \"System Events\" "
```

Comment:
earlier we could set desktop wallpapers individually for mac OS, now its one wallpaper for all desktop, isnt this a step backwards? , although it's fine for now since its reducing requirements, but i think you should open a new issue once merged.